src/screens/TopicPageScreen/index.js

```diff
@@ -101,6 +100,12 @@ const TopicPageScreen = (props) => {
     extrapolate: 'clamp'
   });
 
+  const opacityImage = scrollY.interpolate({
+    inputRange: [0, 100],
+    outputRange: [0, 1],
+    extrapolate: 'clamp'
+  });
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new interpolation for opacityImage has been added. This will control the opacity of an image based on the scroll position. Ensure that this does not conflict with any other animations or effects on the page.
```javascript
const opacityImage = scrollY.interpolate({
  inputRange: [0, 100],
  outputRange: [0, 1],
  extrapolate: 'clamp'
});
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->